streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -561,12 +561,12 @@ public void shouldNotViolateEosIfOneTaskFailsWithState(final String eosConfig, f
                 getMaxPerKey(expectedResultBeforeFailure),
                 "The state store content before failure do not match what expected");
 
-            errorInjected.set(true);
+            //errorInjected.set(true);
```

Comment:
Seems like an invalid change? (btw: this test failed on the CI build -- I assume this unnecessary change is the reason why it's failing?)

Reply:
I was going through the class and playing around it to learn. I will revert it .